docs/src/design/story/wireframes/v1_5/styles.css

```
@@ -0,0 +1,739 @@
+  --ink-3: #6a6a6a;
+  --ink-4: #9a9a9a;
+  --paper: #f6f3ea;
+  --paper-2: #ede9dc;
+  --rule: #c9c3b0;
+  --rule-strong: #2a2a2a;
+  --accent: #b23a1d; /* red pencil */
+  --accent-ink: #5a1b0d;
+  --blue-pencil: #3a5d7a;
+  --grid: #d6cfb8;
+  --grid-minor: #e3dcc7;
+  --mono: "JetBrains Mono", "IBM Plex Mono", ui-monospace, Menlo, monospace;
```

Stylelint (17.11.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)